mono/eglib/gmisc-unix.c

```diff
@@ -30,6 +30,7 @@
 #include <stdlib.h>
 #include <glib.h>
 #include <pthread.h>
+#include "../mono/utils/mono-static-mutex.h"
```

Comment:
The layering here is wrong - we don't make eglib depend on mono/utils.  You should define everything in eglib and use it in utils, metadata, etc.

Reply:
I suspect this is due to bringing back old deleted mono code.

Reply:
I'm not well aware of the eglib vs. mono seperation.
If I understand well, I should put the header in eglib, and use it in mono?